desktop/release-notes.md

```diff
@@ -23,17 +23,86 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+
+### Upgrades
+- [Docker Engine to v20.10.17](https://docs.docker.com/engine/release-notes/#201017)
+- [Docker Compose v2.6.1](https://github.com/docker/compose/releases/tag/v2.6.1)
+- [Kubernetes v1.24.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.1)
+- [cri-dockerd to v0.2.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.1)
+- [CNI plugins to v1.1.1](https://github.com/containernetworking/releases/tag/v1.1.1)
+- [containerd to v1.6.6](https://github.com/containerd/containerd/releases/tag/v1.6.6)
+- [runc to v1.1.2](https://github.com/opencontainers/runc/releases/tag/v1.1.2)
+- [Go 1.18.3](https://github.com/golang/go/releases/tag/go1.18.3)
+
+### New
+
+- User can now add environment variables when running an image right from the Docker Dashboard.
+- Added features to make it easier to work with containers logs such as regex search and the ability to clear container logs while the container is still running.
+- Addressed feedback on new containers tables, added ports and separated container and image names.
+- Two new extensions (Ddosify and Lacework) added to the Extensions Marketplace.
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Containers tab: Bulk actions for start/pause/stop selected containers.
+- Containers tab: Pause/restart actions for compose projects.
+- Containers tab: Adds icons as well as the exposed port(s) / exit code.
+- External URLs can refer to extension details in Docker Desktop Marketplace using links like `docker-desktop://extensions/marketplace?extensionId=docker/logs-explorer-extension`.
+- Implemented changes on the logs search functionality and added the possibility of clearing the terminal.
+- Expanded / collapsed state of the Compose apps is now persisted.
+- `docker extension` CLI commands are available with Docker Desktop by default.
+- Extensions: make screenshots displayed in the marketplace bigger.
+- Fixed a bug where a Docker extension fails to load if its backend container(s) are stopped. Fixes [docker/extensions-sdk#16](https://github.com/docker/extensions-sdk/issues/162).
+- Fixed a bug where image search field was cleared without reason. Fixes [docker/for-win#12738](https://github.com/docker/for-win/issues/12738).
+- Fixed a bug where license agreement is not showing up, thus silently blocking Docker Desktop startup.
+- Fixed the displayed image and tag for unpublished extensions to actually display the ones from the installed unpublished extension.
+- Fixed the duplicate footer on the Support screen.
+- Dev Environments can be created from a subdirectory in a GitHub repository.
+- Occasionally the Docker engine will restart during a `docker system prune`. This is a [known issue](https://github.com/moby/buildkit/pull/2177) in the version of buildkit used in the current engine and will be fixed in future releases.
```

Comment:
This needs to be in a 'Known issue' section and with a 'For all platforms' subheading. Similar to the screenshot 

![image](https://user-images.githubusercontent.com/102604716/176192308-8c06740c-8dea-4bd4-803a-3bb4e2404f98.png)